src/test/java/org/opensearch/jobscheduler/rest/RestGetLockActionIT.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.rest;
+
+import com.carrotsearch.randomizedtesting.annotations.ThreadLeakScope;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import org.junit.Before;
+import org.mockito.Mockito;
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.cluster.service.ClusterService;
+import org.opensearch.common.bytes.BytesArray;
+import org.opensearch.common.xcontent.LoggingDeprecationHandler;
+import org.opensearch.common.xcontent.NamedXContentRegistry;
+import org.opensearch.common.xcontent.XContentBuilder;
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.common.xcontent.XContentParser;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.jobscheduler.JobSchedulerPlugin;
+import org.opensearch.jobscheduler.TestHelpers;
+import org.opensearch.jobscheduler.spi.utils.LockService;
+import org.opensearch.jobscheduler.transport.AcquireLockRequest;
+import org.opensearch.rest.RestHandler;
+import org.opensearch.test.OpenSearchTestCase;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.test.rest.FakeRestChannel;
+import org.opensearch.test.rest.FakeRestRequest;
+
+@ThreadLeakScope(ThreadLeakScope.Scope.NONE)
+public class RestGetLockActionIT extends OpenSearchTestCase {
+
+    private ClusterService clusterService;
+    private LockService lockService;
+    private RestGetLockAction getLockAction;
+    private String getLockPath;
+    private String testJobId;
+    private String testJobIndexName;
+    private long testLockDurationSeconds;
+    private String requestBody;
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        this.clusterService = Mockito.mock(ClusterService.class, Mockito.RETURNS_DEEP_STUBS);
+        Mockito.when(this.clusterService.state().routingTable().hasIndex(".opendistro-job-scheduler-lock")).thenReturn(true);
+        this.lockService = new LockService(Mockito.mock(NodeClient.class), clusterService);
+        this.getLockAction = new RestGetLockAction(this.lockService);
+        this.getLockPath = String.format(Locale.ROOT, "%s/%s", JobSchedulerPlugin.JS_BASE_URI, "_get/_lock");
+        this.testJobId = "testJobId";
+        this.testJobIndexName = "testJobIndexName";
+        this.testLockDurationSeconds = 1L;
+        this.requestBody = "{\"job_id\":\"test\",\"job_index_name\":\"test\",\"lock_duration_seconds\":\"1\"}";
```

Comment:
Having the same name for different things you're testing doesn't help you deconflict whether you assigned them to the wrong things.  Why not use the `testJobId` and `testJobIndexName` values here?  Not that they are ever used, of course, you could probably put any string here and it would work. :)